README.md

```diff
@@ -0,0 +1,11 @@
+# Research project on amplitude models for N-star and other resonances in the GlueX experiment
+
+This repository is under active development.
+
+We are doing amplitude analysis / Partial Wave Analysis (PWA) with the use of symbolic expressions, which is included in the `ComPWA` packages, or CAS(Computer Algebra System)-assisted model building in general (the python package `SymPy` in this case).
```

Comment:
- ComPWA without backticks (code style)
- Same for SymPy, or write `sympy` (the package name)